mct_quantizers/pytorch/quantizers/base_uniform_inferable_quantizer.py

```diff
@@ -41,9 +44,24 @@ def __init__(self,
             """
 
             super(BaseUniformInferableQuantizer, self).__init__()
-            self.num_bits = num_bits
+
+            assert isinstance(min_range, list), f'min_range is expected to be a list, but is of type {type(min_range)}'
+            assert isinstance(max_range, list), f'max_range is expected to be a list, but is of type {type(max_range)}'
+
+            for _min, _max in zip(min_range, max_range):
+                assert _min<_max, f"Max range must be greater than min value but min is {_min} and max is {_max}"
+
+            # Align mix/max numpy arrays so they are torch Tensors on the working device
+            min_range = to_torch_tensor(np.asarray(min_range)).to(get_working_device())
+            max_range = to_torch_tensor(np.asarray(max_range)).to(get_working_device())
+
+            min_range, max_range = fix_range_to_include_zero(min_range,
+                                                             max_range,
+                                                             num_bits)
```

Comment:
Check if you don't have extra call to "fix_range_to_include_zero" in WeightsUniformInferableQuantizer